java/src/jmri/configurexml/LoadXmlConfigAction.java

```diff
@@ -95,24 +98,62 @@ protected boolean loadFile(JFileChooser fileChooser) {
         return results;
     }
 
-    static public java.io.File getFile(JFileChooser fileChooser) {
+    /**
+     * Get the File from a given JFileChooser.
+     * @return the selected File.
+     * @deprecated use {@link #getFile(JFileChooser fileChooser, Component component)}
+     * @param fileChooser the JFileChooser for the file.
+     */
+    @CheckForNull
+    @Deprecated (since="5.7.8",forRemoval=true)
+    public static File getFile(@Nonnull JFileChooser fileChooser) {
+        return getFile(fileChooser, null);
+    }
+
+    /**
+     * Get the File from an Open File JFileChooser.
+     * If a Component is provided, this helps the JFileChooser to not get stuck
+     * behind an Always On Top Window.
+     * @param fileChooser the FileChooser to get from.
+     * @param component a Component within a JFrame / Window / Popup Menu.
```

Comment:
I suggest:
component a Component within a JFrame / Window / Popup Menu`, or the JFrame or Window itself`.